packages/http-server/src/server.ts

```diff
@@ -14,7 +14,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
   const { components, config } = opts;
 
   const server = fastify({
-    logger: (components && components.logger) || createLogger('HTTP SERVER'),
+    logger: components.logger,
```

Comment:
I think we should lean towards having more default parameters. Ideally, a user should be able to spin up the server with just: `createServer([])`

Reply:
The final user will always be able. They use the public client which has everything defaulted so they'll be able to use it with no parameters at all.

On the other hand, this is the internal client that I am using to create the hosted Prism version and this one requires this stuff.

Does that clarify?

Reply:
Yes it does!